webrtc.html

```diff
@@ -2882,7 +2882,8 @@ <h2>Methods</h2>
                     method was invoked.</p>
                   </li>
                   <li>
-                    <p>If both <var>sdpMid</var> and <var>sdpMLineIndex</var> are
+                    <p>If <var>candidate</var> is not an empty string and both
+                    <var>sdpMid</var> and <var>sdpMLineIndex</var> are
```

Comment:
All of these should probably say candidate.foo. The line above is "let candidate be the method's first argument"